superset-frontend/packages/superset-ui-core/src/components/SafeMarkdown.tsx

```diff
@@ -25,15 +25,16 @@ import { FeatureFlag, isFeatureEnabled } from '../utils';
 
 interface SafeMarkdownProps {
   source: string;
-  htmlSanitization?: boolean;
-  htmlSchemaOverrides?: typeof defaultSchema;
 }
 
-function SafeMarkdown({
-  source,
-  htmlSanitization = true,
-  htmlSchemaOverrides = {},
-}: SafeMarkdownProps) {
+function SafeMarkdown({ source }: SafeMarkdownProps) {
+  const appContainer = document.getElementById('app');
```

Comment:
@geido Components in `superset-ui-core` and `src/components` may be extracted in the future to a library that can be used by other apps. We shouldn't create a dependency with Superset DOM to get the configurations because different apps will have different DOM structures.

Reply:
@michael-s-molina the PR is now updated